internal/repos/importer.go

```diff
@@ -169,3 +183,22 @@ func (i *importer) handleGitHubImport(ctx context.Context, imp db.ListRepoImport
 
 	return tx.Commit(ctx)
 }
+
+func handleDeletedRepos(ctx context.Context, tx pgx.Tx, i *importer, repos []*githubRepo, repoOwner string) error {
+	var currentRepos []string
+	var err error
+
+	i.logger.Info().Msg("Starting to delete removed repos")
```

Comment:
```suggestion
	i.logger.Info().Msg("starting to delete removed repos")
```

(lower case just to be consistent for now)